other/date_to_weekday.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+
+
+def date_to_weekday(inp_date: str) -> str:
+    """
+    It returns the day name of the given date string.
+    :param inp_date:
+    :return: String
+    >>> date_to_weekday("7/8/2035")
+    'Tuesday'
+    >>> date_to_weekday("7/8/2021")
+    'Saturday'
+    >>> date_to_weekday("1/1/2021")
+    'Friday'
+    """
+    day_list: list = [
```

Comment:
We have pretty module in python so we don't need to hardcode weekdays `import calendar;calendar.day_name`